### ISSUE TITLE
Implement deep learning algorithm to improve accuracy

### ISSUE THREAD

**Issue by SanjanaMahesh3** [Rakesh9100/ML-Project-Drug-Review-Dataset#50]
### Prerequisites

- [X] I am an Open Source Contributor
- [X] I checked to make sure that this issue has not already been filed

### Description

Most of the time deep learning techniques are better than ML techniques. Maybe trying out a deep learning technique would be better on the dataset. I plan to try Convolution Neural Network(CNN)

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/Rakesh9100/ML-Project-Drug-Review-Dataset/blob/main/.github/CONTRIBUTING_GUIDELINES.md) of this project.

**Comment by Rakesh9100** [comment]
Assigning, proceed @SanjanaMahesh3

**Comment by Rakesh9100** [comment]
@SanjanaMahesh3 Update of your work

**Comment by Tisha6661** [comment]
Please assign me this task @Rakesh9100  under gssoc 23.

**Comment by SanjanaMahesh3** [comment]
@Rakesh9100 I have tried CNN and ANN it's accuracy currently is around 30%. I am trying better feature extractors to improve accuracy

**Comment by SanjanaMahesh3** [comment]
If it is not better I can implement RCNN or Graph based networks too

**Comment by Rakesh9100** [comment]
Okay proceed and give a try @SanjanaMahesh3

**Comment by SRINIVAS-2003** [comment]
Please assign me this task, I would implement deep learning algorithms to improve the accuracy

**Comment by Rakesh9100** [comment]
Update pls @SanjanaMahesh3 

Pls specify the names of the algorithms you will implement @SRINIVAS-2003

**Comment by SRINIVAS-2003** [comment]
I want to implement ANN, AdaBoost algorithms. Some of the major algorithms were already implemented,
so I want to give a try to implement the above deep-learning algorithms.

**Comment by Rakesh9100** [comment]
Proceed @SRINIVAS-2003

**Comment by Rakesh9100** [comment]
Update of your work @SRINIVAS-2003

**Comment by SRINIVAS-2003** [comment]
Yeah sure @Rakesh9100 .
I have implemented Ann and Adaboost algorithms and obtained some of the performance metrics like accuracy, R2 score , mean square error. 
Accuracy obtained is 13% near or less for both the algorithms. 

Now I'm trying to plot the graphs of predictes Vs testing and training data values for the above two algorithms.

**Comment by Rakesh9100** [comment]
Okay @SRINIVAS-2003

**Comment by SRINIVAS-2003** [comment]
@Rakesh9100 
Can I update the code for the above algorithms I implemented in the main.py file 
Accuracies for ANN is 31% and AdaBoost id 13% , yesterday I confused as both are same digits( 31 and 13 not seen properly) so I said 13% for both the algorithms.
Sorry for the inconvenience @Rakesh9100 .

**Comment by Rakesh9100** [comment]
Yes, you can update your code by raising a PR and can also plot the graphs of the accuracy predictions of the models.
Make sure to pull the latest code changes to avoid merge conflicts @SRINIVAS-2003

**Comment by SRINIVAS-2003** [comment]
Yeah sure @Rakesh9100, I will implement accuracy plot also.

**Comment by SRINIVAS-2003** [comment]
@Rakesh9100 I have committed some changes in the main file. After clicking on commit changes it asked to create pull request,I have created a pull request for updation for main.py file.
Am I proceeding in right process? 
I don't have that much touch with the GitHub , so I'm having doubt.
Please kindly help me so that I'll able to handle it.

**Comment by Rakesh9100** [comment]
I am here to help you out, don't worry.
You are proceeding in a ryt way, pls check the comment on your PR @SRINIVAS-2003

**Comment by SRINIVAS-2003** [comment]
Thank you very much for helping me @Rakesh9100 .
I have raised PR using the template which u have provided.

**Comment by SRINIVAS-2003** [comment]
@Rakesh9100 is the previous commit accepted without any problem?
I have done linting also successfully.
